discord/gateway.py

```diff
@@ -553,6 +553,8 @@ def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
         self.max_size = None
         self._keep_alive = None
+        self._ssrc_lookup = {}
+        self._events_to_fire = []
```

Comment:
The gateway doesn't have a ref. to the VoiceClient, so I stop the SSRCs on the Gateway because I have no way to move them to the VoiceClient. Would it be better to give the Gateway a ref. to VoiceClient?